app/src/main/java/com/github/se/eventradar/ui/chat/Chat.kt

```diff
@@ -65,7 +62,8 @@ import java.time.LocalDateTime
 @Composable
 fun ChatScreen(viewModel: ChatViewModel = hiltViewModel(), navigationActions: NavigationActions) {
```

Comment:
I see you've made the factory changes in this PR. In that case, can you modify the UI at the same time?

This line should be:
`fun ChatScreen(viewModel: ChatViewModel, navigationActions: NavigationActions) {
`

I would suggest adding the nav already, but that can be done in the next PR, as we discussed :)

Reply:
I believe this should also be done in the PR that updates Chat.kt to use the VM.